### PR TITLE
Pia 3478 cleanup gini configuration and gini bank configuration options

### PR DESCRIPTION
**Pull request by vizaknaidavid**

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 1]
```diff
@@ -248,19 +248,6 @@ public final class GiniBankConfiguration: NSObject {
      Sets the continue button text in the navigation bar on the review screen.
      */
     @objc public var navigationBarReviewTitleContinueButton = ""
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:250]: I think navigationBarReviewTitleContinueButton can be removed as well 😉

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -81,11 +81,7 @@
 "ginicapture.onboarding.qrCode.title"  = "QR codes supported";
 "ginicapture.onboarding.qrCode.description"  = "Besides invoices, QR codes
 are also supported";
-"ginicapture.review.top"                = "Is the document complete, in focus and in the correct orientation?";
-"ginicapture.review.bottom"             = "Double-tap to check the focus";
-"ginicapture.review.rotateButton"       = "Rotate document";
 "ginicapture.review.documentImageTitle" = "Document";
-"ginicapture.review.unknownError"       = "An unknown error has occurred on the review screen";
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/en.lproj/Localizable.strings:88]: [Suggestion] You need to return back
` "ginicapture.review.unknownError" = "An unknown error has occurred on the review screen";
` 
It stays in German translation and is used above.

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great 👍 😊

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift (hunk 1 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:250]: I will check the button resources that we have on ScreenAPICoordinator, and if they are not used, I will remove all of them

Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:250]: It seems that they are used in Xamarin code, so it should stay :D

**Review by zladzeyka — APPROVED** [review]
👏🏻

**Pull request merged**